create-quasar/templates/app/quasar-v2/ts-vite/BASE/src/components/ExampleComponent.vue

```diff
@@ -12,8 +12,29 @@
   </div>
 </template>
 
-<script lang="ts">
-<% if (typescriptConfig === 'composition') { %>import {
+<% if (typescriptConfig === 'composition-setup') { %><script setup lang="ts">
+import { computed, ref } from 'vue';
+import { Todo, Meta } from './models';
+
+interface Props {
+  title: string;
+  todos?: Todo[];
+  meta: Meta,
```

Comment:
A mistake from my previous suggestion.
```suggestion
  meta: Meta;
```